.gitignore

```
@@ -8,6 +8,7 @@ vendor
 *.dylib
 bin/*
 testbin/*
+hack/tools/bin/*
```

IMO, we should be more explicit in some of these cases. IIRC, leaving the leading slash out means that we will ignore any path under the current dir that ends with `hack/tools/bin/*`. Whereas including the leading slash means _just_ this specific directory from the root (based on the location of the gitignore file). IMO explicitness is almost always better.

There's a reasonable argument one could make about doing this more ubiquitously for all relevant .gitignore paths in a single PR, so this is non-blocking, regardless.

```suggestion
/hack/tools/bin/*
```

---

Fully in agreement here, I'll add your suggestion. Thanks!